tools/check_style_proto.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
We now have two places where we run `clang-format`: in the `jidicula/clang-format-action`  which only acts on C/C++, and in this file which checks only `.proto` files (but could check other file types in the future).

Would it be possible to modify the `find` command below slightly, find not just `.proto` but also `.cc` and `.h`, and check all of our code in just a single place? In other words, could we change this file to be `check_style.sh` (not proto-specific) and retire the use of `jidicula/clang-format-action`?

Reply:
Sure. I will do it tomorrow.